upgrade/scripts/sls/csm_can_to_chn/sls_updates.py

```diff
@@ -87,6 +98,22 @@ def create_bican_network(networks, default_route_network_name):
         bican = BicanNetwork(default_route_network_name=default_route_network_name)
         networks.update({bican.name(): bican})
 
+def delete_can_network(networks):
+    """Delete SLS CAN network data structure."""
+
+    if networks.get("CHN") is None:
+        click.secho("CHN subnet not found in SLS", fg="red")
```

Comment:
Needs better messaging:  `CHN subnet not found in SLS.  To remove the CAN, first create a CHN.`